requirements.txt

```diff
@@ -1,18 +1,25 @@
-click
 # until a stable version with handle_exception is released on pip
 git+https://github.com/matrix-org/matrix-python-sdk.git
 
 git+https://github.com/raiden-network/raiden-contracts.git
-git+https://github.com/raiden-network/raiden-libs.git
+# raiden-contracts==0.8.0
 
-gevent
-jsonschema
+dataclasses==0.6
```

Comment:
Not really related to this PR: Raiden just moved to python3.7. If we follow suit, we won't need this as a dependency, anymore.

Reply:
Good point. The porblem is that there is no python 3.7 environment on travis yet. So either we switch to CircleCI as well ort find another solution, but I agree we should use 3.7 as well if Raiden does it.

Reply:
Opened #53